internal/util/password/scramsha256_test.go

```diff
@@ -43,23 +43,23 @@ type scramSHA256TestCase struct {
 var scramSHA256TestCases = map[string]scramSHA256TestCase{
 	// Test vector generated with db.runCommand({createUser: "user", pwd: "pencil", roles: []})
 	"FromMongoDB": {
-		params: scramSHA256Params{
+		params: scramParams{
 			iterationCount: 15000,
 			saltLen:        28,
 		},
 		password: "pencil",
-		salt:     must.NotFail(base64.StdEncoding.DecodeString("vXan6ZbWmm5i+f+mKY598rnIfoAGGp+G9NP0qQ==")),
+		salt:     must.NotFail(base64.StdEncoding.DecodeString("qhzna/QaH741WENngAzoR1QhHqy9GE/fkyNEMQ==")),
 		want: must.NotFail(types.NewDocument(
-			"storedKey", "bNxFkKtMt93v+ha80yJsDG6Xes3GOMh5qsRzwkcF85s=",
+			"storedKey", "PwtAytvH2feCBa70lRzdg4DOZaeDVHSUrYIYt5dRVog=",
 			"iterationCount", int32(15000),
-			"salt", "vXan6ZbWmm5i+f+mKY598rnIfoAGGp+G9NP0qQ==",
-			"serverKey", "1m33jRKioBEVpJzDdJeG5SgKPEmhPNx3A0jS4fINVyQ=",
+			"salt", "qhzna/QaH741WENngAzoR1QhHqy9GE/fkyNEMQ==",
+			"serverKey", "kZY6FfqXOsAt+qEl6a/R70KoPnEZOtyfkWJzjyiFsmw=",
```

Comment:
What was the reason for the change in hash? I don't see change in the input :thinking:

Reply:
Oops. Reverting. It was mistakenly changed somehow (copied this rather than new values for SHA1 tests). Still with valid data, but it was unintentional, so I am changing back for clarity's sake.